blobs.go

```diff
@@ -103,6 +103,9 @@ type BlobDeleter interface {
 // BlobEnumerator enables iterating over blobs from storage
 type BlobEnumerator interface {
 	Enumerate(ctx context.Context, ingester func(dgst digest.Digest) error) error
+
+	// ModTime returns the modification time of the blob from the storage.
+	ModTime(ctx context.Context, dgst digest.Digest) (time.Time, error)
```

Comment:
It would change [BlobDescriptorService](https://github.com/docker/distribution/blob/1e2f10eb65743fed02f573d31a4587de09afb20e/blobs.go#L112). Do you want to store driver.FileInfo in Redis?